### ISSUE TITLE
Indicate which records have been updated

### ISSUE THREAD

**Issue by StephenCWills** [GridProtectionAlliance/SystemCenter#8]
Some lists in System Center have both `Save Changes` and `Clear Changes` options. It would be nice to know which rows, or even which individual fields, will be committed or reverted when clicking these buttons. This can be done by highlighting edited rows or fields. This would also help prevent errors during data entry.

**Comment by StephenCWills** [comment]
Marked as `wontfix` for now. The user can use `Save Changes` liberally to avoid making large adjustments to configuration where the changes would need to be reviewed before committing.

**Comment by clackner-gpa** [comment]
This has partially been implemented. 
When hovering over the `Clear Changes` button a tooltip appears that indicates what changes will be removed
![image](https://user-images.githubusercontent.com/52460212/117207767-5407f380-adc2-11eb-8d9a-f0209f57f4e7.png)

Is there a need to implement a similar tooltip on the `Save Changes` button?

**Comment by StephenCWills** [comment]
> Is there a need to implement a similar tooltip on the `Save Changes` button?

No, I think that would just be redundant. This is a good step in the right direction. Thanks.

**Comment by clackner-gpa** [comment]
With '7806af8` whis has been fixed in a number of additional places.
